### ISSUE TITLE
Dark theme?

### ISSUE THREAD

**Issue by kkorus** [paolosalvatori/ServiceBusExplorer#299]
Can you add option for Dark Theme?

**Comment by paolosalvatori** [comment]
Hi @kkorus love this idea, any ideas for colors? Any reference or sample?

**Comment by kkorus** [comment]
@paolosalvatori Something like here https://git-fork.com/  It supports light and dark theme.

**Comment by paolosalvatori** [comment]
Do you have bandwidth to implement this feature? I've a very limited spare time in this period :(

**Comment by ErikMogensen** [comment]
@gitricharde is already working on this.

**Comment by DavidSchor** [comment]
Is @gitricharde still working on this or is it currently not being worked on?

**Comment by ErikMogensen** [comment]
As far as I know he is not working on it anymore, nor is anyone else. You are more than welcome to implement it.

**Comment by gitricharde** [comment]
@DavidSchor Please feel free to carry on 👍

**Comment by paolosalvatori** [comment]
I'm closing the issue for now, feel free to reopen it or submit a PR :)

**Comment by DavidSchor** [comment]
Yes, that is probably best. I was hoping to take a look at it during the holiday, but didn't. I am having a bit of crunch time at work now, so it would be a while before I get a chance to look at it.

**Comment by paolosalvatori** [comment]
Take your time @DavidSchor and thanks for the collaboration!

**Comment by SeanFeldman** [comment]
That’d be great, @JohnHallstrom

**Comment by ErikMogensen** [comment]
@JohnHallstrom, in case you missed it, there was a PR previously for this, #306.

**Comment by JohnHallstrom** [comment]
> @JohnHallstrom, in case you missed it, there was a PR previously for this, #306.

Thanks for the heads-up!

**Comment by andremiranda-act3d** [comment]
Hi, everyone!

Is there any news about this? I'd really appreciate having a dark mode on this :-)

**Comment by ErikMogensen** [comment]
I haven't heard anything.

**Comment by plunet** [comment]
Another option if someone has any inertia (I'm not a coder, so can't offer to help) would be to nudge the existing code to better respect re-rendering of the interface by the OS in Windows High Contrast Mode, or Contrast Themes, rather than developing a new dark theme in the app itself. 

Much of the content windows does re-render in a contrast mode but the surrounds don't, the menus and some other things become invisible. For those who really need dark mode for accessibility reasons there's more chance they'll be using one of the Contrast Modes as it usually deals with everything in a fairly consistent manner, whereas individual dark modes are very much a piecemeal approach to the problem of achieving a predominantly dark screen. For someone who knows how apps react to Contrast Themes this might be an easier ask to develop?

Example of how Service Bus Explorer 6.0.3 behaves in high contrast mode below.

<img width="610" alt="Image" src="https://github.com/user-attachments/assets/03fb3a2a-a295-459d-b832-2d90192a5488" />

**Comment by dsiddharth2** [comment]
I will take this up.

Here is a glimpse of some work.

<img width="1410" height="867" alt="Image" src="https://github.com/user-attachments/assets/efd24428-26ce-4b77-ad5b-32cb115cb9a9" />

**Comment by SeanFeldman** [comment]
> I added dark mode support.https://github.com/abukres/ServiceBusExplorer_Darkmode
> 
> Read the top part of the readme for important information.

You'll need to open a PR.

**Comment by SeanFeldman** [comment]
> I added dark mode support.https://github.com/abukres/ServiceBusExplorer_Darkmode
> 
> Read the top part of the readme for important information.

The comment is misleading. Especially the opening sentence. Please update it to indicate it's a partial support.

**Comment by abukres** [comment]
> > I added dark mode support.https://github.com/abukres/ServiceBusExplorer_Darkmode
> > Read the top part of the readme for important information.
> 
> The comment is misleading. Especially the opening sentence. Please update it to indicate it's a partial support.

"The rest of the application has not been converted. I am making it available publicly for your convenience. If you want other UI elements to be converted, an AI assistant can be used to do the conversion work."
This indicates it has partial support. 
It's my repo. Anyone is free to use it or not in the format it is in.
